go.sum

```diff
@@ -275,6 +278,8 @@ github.com/onsi/gomega v1.7.0/go.mod h1:ex+gbHU/CVuBBDIJjb2X0qEXbFg53c61hWP/1Cpa
 github.com/op/go-logging v0.0.0-20160315200505-970db520ece7/go.mod h1:HzydrMdWErDVzsI23lYNej1Htcns9BCg93Dk0bBINWk=
 github.com/openshift-online/ocm-sdk-go v0.1.117 h1:MbRDTDfa1ip7pmgp8zSaE/7X933OuysvcUYDPBWh0bk=
 github.com/openshift-online/ocm-sdk-go v0.1.117/go.mod h1:BBF7bNkfHkkVwKolft9AsbYXJBsFiSOFQ6bLSedQby0=
+github.com/openshift-online/ocm-sdk-go v0.1.122 h1:IooPjb+CWhPICuhSTMLxJsKyLmvVV5GRlHug/QRU0bU=
+github.com/openshift-online/ocm-sdk-go v0.1.122/go.mod h1:3i3a/LEYtC7DMor3N94PTPn1+0qq+Fk+iLjt1Z0WVCs=
```

Comment:
You may want to run `go mod tidy` to drop unused old lines

Reply:
good idea. done